tests/e2e_regression/reindex_and_run.sh

```diff
@@ -7,18 +7,20 @@ set -euo pipefail
 
 cat >/tmp/e2e_config.yaml <<EOF
```

Comment:
Maybe this can be a separate file now 🤔

Reply:
I'm not strongly opposed to it. But I like having it all in one place, and the config is pretty short. The goal of this file is basically to set up a test fixture, and in test code, it's common to just inline the fixtures for readability, as the code tends to be read more times than it's changed. It's a judgement call tho.